txpool/mock.go

```diff
@@ -0,0 +1,56 @@
+package txpool
```

Comment:
Ending a filename with `_test.go` will have go treat the file as if it had tests in it (which is not true for a bunch of files already in the codebase).

I suggest creating ` testing.go` file within the module where all the relevant mock methods/structs and helper methods would be defined.

Reply:
Leaving a file without "_test" suffix will have go compiler treat the file as production code, which is also not true. 
> Ending a filename with _test.go will have go treat the file as if it had tests in it (which is not true for a bunch of files already in the codebase).
I don't understand what's your point here, what are the consequences since the code in this file is actually used only in tests

Reply:
[Renamed](https://github.com/0xPolygon/polygon-sdk/pull/287/commits/36a69b206a25c717e6549dc8949c4f4b069e862b)